### DISCUSSION TITLE
How to disable logging the SentryProfilerState debug backtrace in XCode console?

### DISCUSSION THREAD

**Discussion by Shawon91sust** [getsentry/sentry-cocoa#3746] [Q&A] [answered]
![Screenshot 2024-03-14 at 11 12 09 PM](https://github.com/getsentry/sentry-cocoa/assets/8108535/88d23e27-f8f5-4729-b791-3e9c535ff3f1)

Sentry is printing hundreds of lines of this log that is making it difficult to get actual log for debug purposes. Any step to prevent the sentryprofilerstate debug log to print will be much helpful.

**Comment by philipphofmann** [comment]
You can disable the debug log messages by setting the `option.debug = false`, which is the default value: https://github.com/getsentry/sentry-cocoa/blob/da2f43eec1c5c0fad8534a91bb23da118aecd5d0/Sources/Sentry/Public/SentryOptions.h#L22-L27

**Comment by armcknight** [comment, marked as answer]
This was a `printf` left in by accident so wouldn't have been controllable via `option.debug = false`, and was removed in 8.22.0. Sorry for the inconvenience!

**Reply by Shawon91sust** [reply]
Thanks for the response, because the answer philipphofmann provided was found everywhere but was of no use. Will update the SDK then
